### ISSUE TITLE
zos init was waiting on ?

### ISSUE THREAD

**Issue by despiegk** [threefoldtecharchive/zosctl#13]
- need to know if no need to wait, if VB is there, kill the VB manage
- do some checks if the Zero-OS is there & ok

**Comment by xmonader** [comment]
Fixed in 657e1fd2d96f7b81c331f39012e0dfa79e81a0a1

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:

657e1fd2d96f7b81c331f39012e0dfa79e81a0a1

# Verification steps:

 ./zos init --name=create                                
```
** executing command vboxmanage modifyvm create   --memory=4096
** executing command vboxmanage modifyvm create   --ioapic on
** executing command vboxmanage modifyvm create   --boot1 dvd --boot2 disk
** executing command vboxmanage modifyvm create   --nic1 nat
** executing command vboxmanage modifyvm create   --vrde on 
** executing command vboxmanage modifyvm create   --natpf1 "redis,tcp,,4444,,6379" 
INFO created machine create
INFO preparing zos machine...
PONG
preparing machine..
created zos machine and we are ready.
```
